src/port.c

```diff
@@ -82,6 +82,15 @@ int wfopen(WFILE** f, const char* filename, const char* mode)
 #endif
 }
 
+
+/* If either pread() or pwrite() are missing, use the local versions. */
+#if ((defined(HAVE_DECL_PREAD) && (HAVE_DECL_PREAD == 0)) || \
```

Comment:
Is there a case that `HAVE_DECL_PREAD` or `HAVE_DECL_PWRITE` aren't defined? If so does it make sense in that case to also default to the local version?

Reply:
Fixed that. Thanks.